### ISSUE TITLE
Deposit Accessibility Attestation

### ISSUE THREAD

**Issue by carakey** [osulp/Scholars-Archive#2826]
### Descriptive summary

This work includes adding a new "accessibility attestation" checkbox where a response is required for deposits. 

For work types unrelated to degree completion, the primary responsibility is on the creator to submit accessible documents.   We need to convey this as a requirement for deposits, verify that the user has acknowledged their responsibility, and provide support as needed. 

This checkbox would be added to user-submitted work types that are not related to student graduation requirements. 

### Expected behavior

_Cara will provide a mockup of the checkbox text, but in general:_

1. The checkbox has brief descriptive text with links to policy (requirement) and guidance (support)
2. The depositor may choose between two options in order to deposit their work, but they cannot decline to answer
3. Option 1 attests the content is accessible; the content is deposited and routed normally for review
4. Option 2 is a request for assistance; the content is deposited but an email is sent to initiate a consultation

Email messaging if Option 2 is selected (after completing all deposit steps):

- To depositor, the confirmation email should include "A Library Accessibility Team member will reach out to you via email to help make your work accessible."
- To library.accessibility(at)o.e / copying scholarsarchive(at)o.e "A user, [ONID or email], has indicated they would like an accessibility consultation for their SA@OSU deposit, [link]."

Complications:

- For no. 4, is it possible to re-route for library review if this is a faculty article work that would normally skip review?
- For consultations, either a Library Accessibility Team member (Sagan) would need to have access to works in review, or, repository staff will need to be involved in providing them a copy of the file(s).

### Related work

Somewhat similar to #2392

**Comment by carakey** [comment]
> LOL I was wondering the same thing. [@carakey](https://github.com/carakey) can you help?

The degree completion or not (i.e. including it on Undergrad Theses and Grad Projects) was a point of distinction when we met with General Counsel. BUT, I don't see anything super decisive in my notes, and my workflow proposal doc has a big question mark on those two types. Let's go ahead and put the attestation on these two for now but I will revisit with Sagan.

**Comment by carakey** [comment]
Right. UT and GP are in a gray area. We asked the lawyers whether we could make non-degree-required deposits be attested, and they said yes that's fine. They didn't say that it's not fine to have :degree-supporting deposits that aren't subject to special review, i.e. these two types: also be attested, at least not as far as anyone remembers... And we don't have a workflow that would support an accessibility review for them. 

All that for the record, and it's a long way of saying: do what Sarah said and put the attestation on these two types (as well as everything else that isn't HCT and GTD).

**Comment by simholt** [comment]
I checked this on all worktypes and they were all good to go except:

**Broken message - Dataset**
**No messages sent - PurchasedEResource**

**Comment by carakey** [comment]
But wait! There's a typo in the message on the submission form. The first link, for OSU's Digital Accessibility Policy, is missing the "y" on "Policy"
